editor/js/Sidebar.Settings.Shortcuts.js

```diff
@@ -89,13 +89,13 @@ Sidebar.Settings.Shortcuts = function ( editor ) {
 
 		switch ( event.key.toLowerCase() ) {
 
-			case 'Backspace':
+			case 'backspace':
 
 				event.preventDefault(); // prevent browser back
 
-				break;
+				// fall-through
```

Comment:
Oh, sorry, I thought it was a bug.

Reply:
No worries! I should have thrown in a comment when I noticed that in the initial code.